commandline/src/main/java/tech/pegasys/web3signer/commandline/PicoCliSlashingProtectionParameters.java

```diff
@@ -115,6 +115,13 @@ public class PicoCliSlashingProtectionParameters implements SlashingProtectionPa
       arity = "1")
   private long dbHealthCheckIntervalMilliseconds = 30000;
 
+  @Option(
+      names = "--Xslashing-protection-db-connection-pool-enabled",
+      description =
+          "Set to false to disable internal database connection pooling. (Default: ${DEFAULT-VALUE})",
+      paramLabel = "<BOOL>")
+  private boolean connectionPoolEnabled = true;
```

Comment:
nit: dbConnectionPoolEnabled to match the option name

Reply:
refactored.